utils/getWords.go

```diff
@@ -0,0 +1,24 @@
+package utils
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+)
+
+func GetWords(n int) []string {
+	var resp, err = http.Get(WORDS_SOURCE + strconv.Itoa(n))
```

Comment:
well, technically, this should be in it's own package (but don't redo this just out of this comment)